modules/api/src/main/java/org/eclipse/xpanse/api/interceptor/LogInterceptor.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api.interceptor;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.util.UUID;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.MDC;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+/**
+ * Custom log interceptor: each link, thread dimension, and add the final link Trace_ID.
+ */
+public class LogInterceptor implements HandlerInterceptor {
+
+    private static final String TRACE_ID = "TRACE_ID";
+
+    /**
+     * Front interceptor.
+     *
+     * @param request  http request
+     * @param response http response
+     * @param handler  http handler
+     * @return true
+     */
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response,
+            Object handler) {
+        String tid = UUID.randomUUID().toString().replace("-", "");
```

Comment:
we are still creating a new UUID here?

Reply:
This UUID can be used to link the http request，if not to output in log，I will remove related code